fuel-core/src/schema/tx/input.rs

```diff
@@ -218,12 +213,15 @@ impl From<&fuel_tx::Input> for Input {
             } => Input::Message(InputMessage {
                 message_id: MessageId(*message_id),
                 sender: Address(*sender),
-                recipient: Address(*recipient),
+                recipient: Address(*owner),
```

Comment:
Again, remove `owner` from `fuel_tx::Input::MessageSigned` and `fuel_tx::Input::MessagePredicate` too. It should just be a simple conversion to `Input::Message`